lib/khttp/ktest/network.go

```diff
@@ -0,0 +1,45 @@
+// +build !release
```

Comment:
This file has some very helpful functions for testing, but are not necessarily for testing the khttp package only.
I don't feel very strongly here, as they are at least related to networking, and generally testing a server, so `khttp/ktest` may be ok. But I wouldn't object a top level `lib/testing` or `lib/ktesting`, with generic testing functions that are useful across other packages.

Reply:
I agree,  will refactor to knetworking package